### ISSUE TITLE
[Bug] UiWindow stop bounding ui elements when out off bound

### ISSUE THREAD

**Issue by Please-Pick-a-Name** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#477]
### Describe the bug

```lua
	UiTranslate(math.sin(GetTime())*300, 0)
	UiRect(1000, 1)
	UiRect(1, 1000)
	UiWindow(300, 300, true,true)
	UiTranslate(300, 0)
	UiColor(1,0,0)
	UiRect(500, 300)
```
this also caused my UIBlur() to leak out to the entire ui

### Steps to reproduce the behavior

1. 

### Expected behavior

no ui element leak/blur leak

### Environment

- Teardown Version: 1.5.2
- OS: Windows 10
- GPU:
- Graphics driver version:
- CPU:

**Comment by Gregory-Gregory** [comment]
Hi @Please-Pick-a-Name,

Unfortunately, the QA team was unable to reproduce the issue. 
Could you please check the issue again in the latest update?

If the issue persists, please share more detailed steps to reproduce the behavior.

Thank you for your cooperation!

**Comment by YuLun-bili** [comment]
Possibly already fixed in 1.5.3
> [Fixed UiWindow content clipping issue when UiWindow is outside screen boundaries.](https://[teardowngame.com/changelog/](https://teardowngame.com/changelog/))

**Comment by Gregory-Gregory** [comment]
@YuLun-bili, thank you for the clarification.

@Please-Pick-a-Name, if you encounter this issue again, please reopen this discussion.
